SusyCommon/XaodAnalysis.h

```diff
@@ -325,6 +341,7 @@ namespace Susy {
     bool matchTruthJet(int iJet); ///< Match a reco jet to a truth jet
 
     bool eleIsOfType(const xAOD::Electron &in, eleID id=eleID::LooseLLH);
+    bool muIsOfType(const xAOD::Muon &in, muID id=muID::Medium);
```

Comment:
We should drop the default argument `id` from both `eleIsOfType()` and `muIsOfType()`.
This parameter is required and if the caller doesn't specify it then one doesn't know what
one is doing. Better being explicit.